### PR TITLE
Update stacks.toml

### PR DESCRIPTION
**Pull request by paradigma-cl**
Added CrossCheck  subsystem and repo

### CODE REVIEW

**Comment by paradigma-cl** [comment]
I do not know what to do with this error.

**Comment by jubos** [comment]
image

You need to create the CrossCheck sub ecosystem.

A good rule of thumb for whether something should be its own ecosystem is if it has its own website, community (discord, telegram), twitter account, and github organization.

File: data/ecosystems/s/stacks.toml [hunk 1]
```diff
@@ -5,6 +5,7 @@ sub_ecosystems = [
   "alexgo-io",
   "arkadiko-dao",
   "citycoins",
+  "CrossCheck",
```
Comment (jubos) [inline data/ecosystems/s/stacks.toml:8]: This needs to have an accompanying toml file

**Review by jubos — CHANGES_REQUESTED** [review]

File: data/ecosystems/s/stacks.toml (hunk 1 above)
Comment (wileyj) [reply data/ecosystems/s/stacks.toml:8]: I'll merge this and fix into #381

**Comment by wileyj** [comment]
this can be closed

**Comment by wileyj** [comment]
this can be closed per #381

**Comment by jubos** [comment]
Closing out in favor of #381.

**Pull request closed without merging**